package.json

```diff
@@ -116,7 +116,7 @@
     "terser-webpack-plugin": "^5.1.4",
     "tiny-glob": "^0.2.9",
     "tippy.js": "^6.2.6",
-    "vue": "2.6.14",
+    "vue": "^2.6.14",
```

Comment:
i don't know if this will or will not break anything... like the styleguide build, the lab build, or the production build... let's hope not i guess

Reply:
wait no, this will def break, you need to make sure that the `vue` version and the `vue-template-compiler` versions are the same, please update the latter as well

Reply:
Oh good catch, thanks.